powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/PowerAuthController.java

```diff
@@ -433,6 +433,43 @@ public ObjectResponse<SupportApplicationVersionResponse> supportApplicationVersi
         return new ObjectResponse<>("OK", powerAuthService.supportApplicationVersion(request.getRequestObject()));
     }
 
+    /**
+     * Call {@link PowerAuthService#getApplicationConfig(GetApplicationConfigRequest)} method and
+     * return the response.
+     * @param request Get application configuration.
+     * @return Application configuration response.
+     * @throws Exception In case the service throws exception.
+     */
+    @PostMapping("/application/config/detail")
+    public ObjectResponse<GetApplicationConfigResponse> getApplicationConfig(@RequestBody ObjectRequest<GetApplicationConfigRequest> request) throws Exception {
+        return new ObjectResponse<>("OK", powerAuthService.getApplicationConfig(request.getRequestObject()));
```

Comment:
The constructor with object value only adds implicitly the status `OK`

```suggestion
        return new ObjectResponse<>(powerAuthService.getApplicationConfig(request.getRequestObject()));
```